charts/console/templates/_helpers.tpl

```diff
@@ -133,6 +205,17 @@ conduktor: invalid database configuration
 {{- end -}}
 {{- end -}}
 
+{{- define "conduktor.validateValues.monitoring" -}}
+{{- if (.Values.config.monitoring).storage -}}
+{{- with .Values.config.monitoring.storage }}
+{{- if or (or .s3 .gcs) (or .azure .swift) -}}
```

Comment:
Maybe we don't need to check for the provider, and just cancel usage of `.storage` as a whole

Reply:
Currently, on values `monitoringConfig.storage` is not empty but each provider configuration are to break down properties documentation. 
It's not a big of a pain to do this validation like that currently so I suggest we keep it like that for now and see if there is some usage issue un future.